src/lpmd_hfi.c

```diff
@@ -188,6 +188,9 @@ static void update_one_cpu(struct perf_cap *perf_cap)
 	if (perf_cap->cpu < 0)
 		return;
 
+	if (!perf_cap->cpu)
+		reset_cpus (CPUMASK_HFI);
```

Comment:
so here we assume
1, HFI table always start with CPU0
2. as a result, the netlink event always start with CPU0
right?